packages/common.vm/tools/vm.common/vm.common.psm1

```diff
@@ -1762,10 +1762,18 @@ function VM-Install-With-Pip {
         [Parameter(Mandatory=$true)]
         [string] $category,
         [Parameter(Mandatory=$false)]
+        [string] $version = "",
+        [Parameter(Mandatory=$false)]
         [string] $arguments = "--help"
     )
     try {
-        VM-Pip-Install $toolName
+        # Conditionally add version string
+        $toolNameForPip = $toolName
+        if (![string]::IsNullOrEmpty($version)) {  # Check if $version is not empty or null
+            $toolNameForPip += "==$version"
```

Comment:
I think it would be more flexible to expect the version to include the `==`:
```suggestion
            $toolNameForPip += "$version"
```

This allows us more flexibility in case we need for example to specify a range of versions. I would then also add a comment next to the `version` argument to document this:
```PS
[string] $version = "",  # Version using pip format, example: "==0.5.0"